doc/whats_new.rst

```diff
@@ -1569,3 +1571,5 @@ of commits):
 .. _Natalie Klein: http://www.stat.cmu.edu/people/students/neklein
 
 .. _Jon Houck: http://www.unm.edu/~jhouck/
+
+.. _Pablo-Arias: https://github.com/Pablo-Arias
```

Comment:
@Pablo-Arias I added your github account here. Feel free to update it to something else if you feel like it.